examples/apps/attributable-map/src/modelContainerRuntimeFactoryWithAttribution.ts

```diff
@@ -18,6 +18,7 @@ import {
 	IProvideRuntimeAttributor,
 } from "@fluid-experimental/attributor";
 import { FluidObject } from "@fluidframework/core-interfaces";
+// eslint-disable-next-line import/no-deprecated
```

Comment:
Do we have a task open to clean all these examples up?

Reply:
[AB#4990](https://dev.azure.com/fluidframework/internal/_workitems/edit/4990)